auto/parse_output.rb

```diff
@@ -131,10 +131,11 @@ def prepare_fixture_line(line)
   def test_passed_unity_fixture(array)
     class_name = array[0]
     test_name  = array[1]
+    test_time = get_test_time(array[array.length - 1])
```

Comment:
I assume this needs to be enabled by something, as by default we do not have the timing information.

Reply:
Thank you for reviewing :)
Yes indeed, the timing output has to be enabled using `UNITY_INCLUDE_EXEC_TIME` flag

Reply:
so shouldnt there be an if statement, otherwise you are just accessing test_name in the worst case?

Reply:
Yes, if the execution time is not present get_test_time will receive a string that wont match the  "( ms)" pattern and return 0 (that strategy was already applied for normal unity output)